helm-chart/kuberay-operator/templates/deployment.yaml

```diff
@@ -61,6 +61,18 @@ spec:
             {{- $argList = append $argList "--watch-namespace" -}}
             {{- $argList = append $argList $watchNamespace -}}
             {{- end -}}
+            {{- if .Values.logging.file -}}
+            {{- $argList = append $argList "--log-file" .Values.logging.file -}}
```

Comment:
The extra `.Values.logging.file` appears to be unnecessary in this line

Reply:
thanks, fixed!